src/lib/publish.js

```diff
@@ -38,7 +42,23 @@ async function publishBatch({ queue, messages, logger }) {
   }
 }
 
+/**
+ * publish a message to a notification topic
+ * @param topic {string} - topic to publish message to, e.g. an SNS ARN
```

Comment:
not sure this format is a valid jsdoc declaration

Reply:
thx, fixed!